### ISSUE TITLE
Material News page mobile view text not reflowing

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5127]
from a11y testing, on  Master at https://github.com/openlibhums/janeway/commit/f23d166d8ebf96fe4875e49b80d3545b7b577920

Material text generally reflows for different screen widths, except there is a problem on narrow widths of the news item page where this doesn't work.

Reflow image on left : Not Reflowing image on Right

<img width="350" alt="Image" src="https://github.com/user-attachments/assets/b69f76e2-d9fc-41c9-9688-5edc2a9ff320" />

<img width="300" alt="Image" src="https://github.com/user-attachments/assets/17be8f44-c07f-459c-b3b5-fd07dae99d32" />

The threshold seems to be a screen width of 600px.

**Comment by mauromsl** [comment]
closed by #5535
